index.html

```diff
@@ -233,7 +233,32 @@ <h2 class="header landing__header header__about">
       </div>
     </div>
   </div>
-
   <!-- end of works -->
+
+  <!-- Contact Form  -->
+  <footer class="contact">
+    <h2 class="contact__header">Contact me</h2>
+    <p class="contact__text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+    <form action="https://formspree.io/f/mdobpebd" method="post" class="contact__form">
+      <ul class="contact__form-list">
+        <li class="contact__form-list--item">
+          <label for="name"></label>
+          <input type="text" id="name" class="contact__form-input" maxlength="30" placeholder="Sirri" required>
+        </li>
+        <li class="contact__form-list--item">
+          <label for="email"></label>
+          <input type="email" id="email" class="contact__form-input" placeholder="example@gmail.com" required>
+        </li>
+        <li class="contact__form-list--item">
+          <label for="message"></label>
+          <textarea name="message" id="message" cols="30" rows="10" class="contact__form-textarea" maxlength="500" placeholder="write a message here"></textarea>
+        </li>
+
+        <li class="contact__form-list--item">
+          <button type="submit" class="contact__form-button">Get in touch</button>
```

Comment:
you forgot to add style to submit button, I suggest styling the button as in the Figma design, and also don't forget to implement the button effect on hover, pressed

- [ ] Your button

![image](https://user-images.githubusercontent.com/61790587/165367854-0bb4c572-6925-4e7b-a290-58855ce5fc89.png)

**Figma Design**
![image](https://user-images.githubusercontent.com/61790587/165367996-f74a71ca-3100-4530-9a01-e11eec90d7dc.png)

Reply:
Hi @youmari, thanks for your detailed review. The above changes have been implimented